### DISCUSSION TITLE
Why showing ads

### DISCUSSION THREAD

**Discussion by processor286** [darkreader/darkreader#12410] [Q&A]
The extension page in Mozilla shows text:

"Dark Reader doesn't show ads"

But I got shown ads when I installed it, and again when I uninstalled it.

Am I misunderstanding what this statement means?

**Comment by envico801** [comment]
I think it is because the extension has sponsors and maybe in the conditions they are obliged to show these ads.

Maybe at the beginning they didn't show ads and now they do and they forgot to update the description of the extension.

**Reply by ELLIOTTCABLE** [reply]
I … wonder if you two both got infected versions of the extension, somehow? That's very strange; there's definitely no ads in the version I have installed. wtf

**Comment by 0xTheFr34k** [comment]
i got “Personal Offer for You” Pop-up when i install it  and the link in the pop-up was a bot net url flagged by virustotal  (link to the analysis)[https://www.virustotal.com/gui/url/ac04535b3b4e1888af93c6afc80f18cb3a1b53b453dd957b546e2fb081593b81]

**Reply by alexanderby** [reply]
Check the extension ID `eimadpbcbfnmbkopoojfekhnkhdbieeh` for Chrome. Some people install malware through Foxified extension (like apps for downloading videos that have absolutely no securify verification).

**Comment by Myshor** [comment]
Tried and only popup I had was https://darkreader.org/help/en/ when I installed extension and https://darkreader.org/goodluck/ when I uninstalled it.
